### PR TITLE
adding sdk_token to workflow run schema

### PR DESCRIPTION
**Pull request by HoneyryderChuck**
"sdk_token" exposed the studio sdk token for v3.6 API.

### CODE REVIEW

File: schemas/workflow_runs/definitions.yaml [hunk 1]
```diff
@@ -21,6 +21,9 @@ workflow_run_shared:
         type: string
         minLength: 1
         maxLength: 128
+    sdk_token:
+      type: ["null", string]
```
Comment (dvacca-onfido) [inline schemas/workflow_runs/definitions.yaml:25]: Please use the nullable construct instead:
```
# Correct
type: integer
nullable: true
# Incorrect
type: null
# Incorrect as well
type:
  - integer
  - null
```

https://swagger.io/docs/specification/data-models/data-types/

File: schemas/workflow_runs/definitions.yaml [hunk 2]
```diff
@@ -21,6 +21,9 @@ workflow_run_shared:
         type: string
         minLength: 1
         maxLength: 128
+    sdk_token:
```
Comment (dvacca-onfido) [inline schemas/workflow_runs/definitions.yaml:24]: I guess this should be added in the `workflow_run_response` object, as it's not a field user can provide.

Comment (HoneyryderChuck) [reply schemas/workflow_runs/definitions.yaml:24]: done

File: schemas/workflow_runs/definitions.yaml [hunk 3]
```diff
@@ -21,6 +21,10 @@ workflow_run_shared:
...
+      type: string
+      nullable: true
+      description: client token to use when loading this workflow run in the Onfido SDK.
```
Comment (dvacca-onfido) [inline schemas/workflow_runs/definitions.yaml:27]: Just for consistence:
```suggestion
      description: Client token to use when loading this workflow run in the Onfido SDK.
```

File: schemas/workflow_runs/definitions.yaml (hunk 2 above)
Comment (dvacca-onfido) [reply schemas/workflow_runs/definitions.yaml:24]: As discussed, this property should be in the `workflow_run_response` section but let's keep this new property aside for the time being.

**Review by nuno-santos-onfido — APPROVED** [review]

**Comment by dvacca-onfido** [comment]
Change LGTM but we'll wait to have that in public documentation as well before merge (and release).

**Review by DavidMealha-Onfido — APPROVED** [review]

**Review by RiccardoPetteruti-Onfido — APPROVED** [review]

**Pull request merged**